build-system/lazy-build.js

```diff
@@ -76,3 +79,14 @@ exports.lazyBuildJs = async function(req, res, next) {
   const matcher = /\/.*\/([^\/]*\.js)/;
   await lazyBuild(req.url, matcher, jsBundles, doBuildJs, next);
 };
+
+exports.preBuildSomeExtensions = function(argv) {
+  const extensions = getExtensionsToBuild(argv);
+  log(green('Pre-building extensions:'), cyan(extensions.join(', ')));
+  for (const extensionBundle in extensionBundles) {
+    const extension = extensionBundles[extensionBundle].name;
+    if (extensions.includes(extension) && !extensionBundle.endsWith('latest')) {
+      build(extensionBundles, extensionBundle, doBuildExtension);
```

Comment:
It would be good to document what is going on here such as why the promise isn't awaited.

Reply:
Done.